StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RightAccessoryButton.swift

```diff
@@ -16,29 +16,34 @@ extension RowButton {
         enum AccessoryType: Equatable {
             case edit
             case viewMore
+            case update
 
             var text: String? {
                 switch self {
                 case .edit:
                     return .Localized.edit
                 case .viewMore:
                     return .Localized.view_more
+                case .update:
+                    return .Localized.update_card
                 }
             }
 
             var accessoryImage: UIImage? {
                 switch self {
                 case .edit:
                     return nil
-                case .viewMore:
+                case .viewMore, .update:
                     return Image.icon_chevron_right.makeImage(template: true).withAlignmentRectInsets(UIEdgeInsets(top: -2, left: 0, bottom: 0, right: 0))
                 }
             }
         }
 
         private var label: UILabel {
             let label = UILabel()
-            label.text = accessoryType.text
+            if accessoryType != .update {
+                label.text = accessoryType.text
+            }
```

Comment:
I don't understand why we have `.update` returning .Localized.update_card for it's text but then not setting the text? Why not just have nil for the text?

Reply:
I could have it return nil, but the accessibility label text is set to .update_card

Reply:
I don't think this should have an accessibility label, it's only really an indicator, not a button like "View more" and "Edit"

Reply:
That could be valid, my UI test just wouldn't be able to find the buttons by identifier the way it did with CircularButton.Edit or Update card. I can still make the change though

Reply:
I think you can update your tests to look for the `accessoryImage` instead, you can see an example of looking for the image in `testSingleCardCBC_update_and_remove_selectStateApplePay`, then have nil text here.

Reply:
The image for the chevron doesn't show up in app.images. I've implemented a workaround that sets the accessibility identifier to "chevron" instead